### PR TITLE
more reasonable genetics nerf

### PR DESCRIPTION
**Pull request by boodaliboo**
alternative to pr https://github.com/yogstation13/Yogstation/pull/10224
i want this pr as gamers is fucking shit and out of touch.
hulk is incredibly powerful 
thermal can fuck over any antag
space adaptation i feel is self explanatory
telekenisis i feel is pretty self explanatory
hulk changed to 60
thermal changed to 35
space adaptation changed to 40
telekenesis changed to 40

#### Changelog

:cl:  
tweak: tweaked genetics instability
/:cl:

### CODE REVIEW

**Comment by jcatjmeow** [comment]
### Intent of your Pull Request
Explain what the purpose of your PR is. Mention if you tested your changes...

### Why is this good for the game?
Explain why each change in your pr is beneficial to the server. This is less necessary for bugfixes and such, but is always welcome. Remember: something that is self evident to you might not be to others. Explain your rationale even if you feel it goes without saying.

**Comment by Marmio64** [comment]
increase SA to 45 so that it cant be combo'd with hulk unless you get a stabilizer chromosome

**Comment by athena413** [comment]
space adapt needs a more substantial cost imo because its basically a free space suit except not even a space suit so you can wear armour and other shit with no penalty

**Comment by MegaEmpirical** [comment]
I feel like your own description is an argument against your PR. I feel so confused.

**Comment by boodaliboo** [comment]
> I feel like your own description is an argument against your PR. I feel so confused.

look at gamers pr and see his nerfs there ludicrous and it also nerfs mutations that are awful anyways like shock touch

**Comment by ToGWtF** [comment]
Thermals is literally the reason gamer made his PR. make it more than 35. its literally round ending for every antag type

File: code/datums/mutations/hulk.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@
 	text_gain_indication = "<span class='notice'>Your muscles hurt!</span>"
 	species_allowed = list("human") //no skeleton/lizard hulk
 	health_req = 25
-	instability = 40
+	instability = 60
```
Comment (JamieD1) [inline code/datums/mutations/hulk.dm]:
```suggestion
	instability = 65
```

File: code/datums/mutations/sight.dm [hunk 2]
```diff
@@ -42,7 +42,7 @@
 	text_gain_indication = "<span class='notice'>You can see the heat rising off of your skin...</span>"
 	time_coeff = 2
 	locked = TRUE
-	instability = 25
+	instability = 35
```
Comment (JamieD1) [inline code/datums/mutations/sight.dm]:
```suggestion
	instability = 40
```

**Review by JamieD1 — CHANGES_REQUESTED** [review]

**Comment by MegaEmpirical** [comment]
Bastards.

**Pull request merged**